Files/View Models/SelectedItemsPropertiesViewModel.cs

```diff
@@ -332,7 +331,6 @@ public async Task GetPropertiesAsync(CancellationTokenSource _tokenSource)
                 catch (Exception ex)
                 {
                     NLog.LogManager.GetCurrentClassLogger().Error(ex, ex.Message);
-                    ItemMD5HashProgress.Value = ItemMD5HashProgress.Maximum;
```

Comment:
If ShowError is true, the color of ProgressBar will become yellow to indicate that there're errors occurred.
However, if the value of ProgressBar is 0, user won't know that the progress of calculation failed because nothing becomes yellow.